### DISCUSSION TITLE
vNet Flowlogs - Are you using them on any level in your Enterprise Scale?

### DISCUSSION THREAD

**Discussion by MikaelJcSoderberg** [Azure/Enterprise-Scale#1758] [General] [answered]
"Configure virtual network to enable Flow Log and Traffic Analytics"
/providers/Microsoft.Authorization/policyDefinitions/3e9965dc-cc13-47ca-8259-a4252fd0cf7b

I was thinking on enabling it on the LandingZones Management group.
I have only used them inside of specific landingzone before(Not though policy)

**Comment by jtracey93** [comment, marked as answer]
Hey @MikaelJcSoderberg, this is on our backlog to enable in the ALZ implementation options. to replace NSG flow logs.

Tagging @sebassem who looks after the prioritisation of all new products/services/features that we have in the backlog for ALZ

**Reply by sebassem** [reply]
Thanks @MikaelJcSoderberg for surfacing this, please monitor our [ALZ roadmap](https://github.com/orgs/Azure/projects/487) as we update it regularly with what's coming and what we are currently working on
